### DISCUSSION TITLE
Zookeeper Image

### DISCUSSION THREAD

**Discussion by d3bt3ch** [strimzi/strimzi-kafka-operator#8206] [General]
What `ZooKeeper` image is used by the `Strimzi` operator. Does the `Strimzi` operator also manages the `ZooKeeper` cluster? Can we use an existing `ZooKeeper` instance with Kafka CRD?

**Comment by scholzj** [comment]
Strimzi always manages its own ZooKeeper cluster that is dedicated to a given Kafka cluster. So you cannot bring your own ZooKeeper or use Strimzi to manage ZooKeeper for some of your other applications. 

It is using its own container image - basically, we use our own Kafka image as the Kafka binaries contain ZooKeeper as part of them.

**Reply by d3bt3ch** [reply]
@scholzj Thanks. Also, which means even if we have two `Strimzi` managed `Kafka` clusters, there would two `Zookeeper` clusters. Even in such a situation no two `Kafka` cluster can share a single `Zookeeper` cluster managed by `Strimzi`?

**Reply by scholzj** [reply]
Correct. Each Kafka cluster has its own ZooKeeper cluster.

**Reply by d3bt3ch** [reply]
Thanks for resolving my query. Does the default `Kafka` image also comes with Zookeeper as part of them?
